lib/shared_helpers.rb

```diff
@@ -15,16 +25,23 @@ def self.in_a_forked_process
     read, write = IO.pipe
 
     pid = fork do
-      read.close
-      result = yield
-      Marshal.dump(result, write)
-      exit!(0) # skips exit handlers.
+      begin
+        read.close
+        result = yield
+      rescue => error
+        result = { _error_class: error.class }
```

Comment:
Sadly we can't `Marshal.dump` the whole error (`Marshal` raises a `TypeError` if we try).